egs/wsj/s5/steps/lmrescore_rnnlm_lat.sh

```diff
@@ -61,6 +61,11 @@ if [ "$rnnlm_ver" == "tensorflow" ]; then
   first_arg="$rnnlm_dir/unk.probs $rnnlm_dir/wordlist.rnn.final"
 fi
 
+if [ "$rnnlm_ver" == "kaldirnnlm" ]; then
```

Comment:
I think it might be better at this point to have a separate script for this type of lattice rescoring, located in scripts/rnnlm/.  That will keep things separate and will make it easier to refactor in future.